websocket/controller.hh

```diff
@@ -17,11 +18,11 @@ constexpr const char* version = "RFC 6455";
 
 class controller : public protocol_server {
     sharded<experimental::websocket::server> _server;
-    bool _active;
+    sharded<cql_transport::cql_server>& _cql_server;
+    bool _active = false;
     std::vector<socket_address> _listen_addresses;
 public:
-    controller();
-    ~controller();
+    controller(sharded<cql_transport::cql_server>& cql_server) : _cql_server(cql_server) {}
```

Comment:
Nit: why not keep out-of-line as before?
Could keep destructor too (or just not add in the first place).